docker/docker-entrypoint.sh

```diff
@@ -132,6 +130,8 @@ _main() {
         set -- dolt sql-server --host=0.0.0.0 --port=3306 "$@"
     fi
 
+    cd $CONTAINER_DATA_DIR
```

Comment:
`dolt sql` and `dolt sql-server` are need to use the same directory, so I set it here